sdks/python/apache_beam/utils/pipeline_options_validator.py

```diff
@@ -168,3 +176,24 @@ def validate_optional_argument_positive(self, view, arg_name):
     if arg is not None and int(arg) <= 0:
       return self._validate_error(self.ERR_INVALID_NOT_POSITIVE, arg, arg_name)
     return []
+
+  def validate_test_matcher(self, view, arg_name):
+    """Validates that on_success_matcher argument (if set) is unpicklable and
+    is instance of hamcrest.core.base_matcher.BaseMatcher"""
+    from apache_beam.internal import pickler
```

Comment:
both of those two imports? Since hamcrest dependency is only installed for test. If moving hamcrest to top, need to add hamcrest in REQUIRED_PACKAGES in setup.py.